extensions/2.0/Khronos/KHR_materials_diffuse_transmission/README.md

```diff
@@ -0,0 +1,241 @@
+# KHR\_materials\_diffuse_transmission
+
+## Contributors
+
+* Tobias Haeussler, Dassault Systèmes [@proog128](https://github.com/proog128)
+* Bastian Sdorra, Dassault Systèmes [@bsdorra](https://github.com/bsdorra)
+* Ed Mackey, AGI [@emackey](https://twitter.com/emackey)
+
+TODO
+## Acknowledgments
+
+TODO
+
+## Status
+
+Experimental
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension adds a Lambertian diffuse transmission BSDF to the metallic-roughness material. Thin, dielectric objects like leaves or paper diffusely transmit incoming light to the opposite side of the surface. For optically thick media (volumes) with short scattering distances and therefore dense scattering behavior, a diffuse transmission lobe is a phenomenological plausible and cheap approximation.
+
+<figure style="text-align:center">
+<p float="left">
+<img src="figures/leaves.jpg" height="350" />
+<img src="figures/thin_translucent.jpg" height="350" />
+</p>
+<sub><figcaption><em>(Thin) translucent leaves/foliage</em></figcaption></sub>
+</figure>
+
+## Extending Materials
+
+The effect is activated by adding the `KHR_materials_diffuse_transmission` extension to any glTF material.
+
+```json
+{
+    "materials": [
+        {
+            "extensions": {
+                "KHR_materials_diffuse_transmission": {
+                    "diffuseTransmissionFactor": 0.25,
+                    "diffuseTransmissionTexture": {
+                      "index": 0
+                    },
+                    "diffuseTransmissionColorFactor": [
+                      1.0,
+                      0.9,
+                      0.85
+                    ]
+                }
+            }
+        }
+    ]
+}
+```
+
+|                                     | Type                                                                | Description                                                                                                                                                                                         | Required                 |
+| ----------------------------------- | ------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------ |
+| **diffuseTransmissionFactor**       | `number`                                                            | The percentage of reflected, non-specularly reflected light that is transmitted through the surface via the Lambertian diffuse transmission, i.e., the strength of the diffuse transmission effect. | No, default: `0`         |
```

Comment:
`The percentage of reflected, non-specularly reflected....` 
 I think this is vague and somewhat misleading.
 
 I propose:
 "The percentage of non-specularly reflected....."

Reply:
I also propose to drop `Lambertian diffuse` since implementations may use some other model.


So it becomes:
`The percentage of non-specularly reflected light that is transmitted through the surface and re-emitted on the other side, i.e., the diffuse transmission effect.`